tldw_Server_API/app/core/AuthNZ/migrations.py

```
@@ -494,6 +494,147 @@ def migration_011_add_enhanced_auth_tables(conn: sqlite3.Connection) -> None:
+        f"""
+        CREATE TABLE IF NOT EXISTS api_keys_new (
+            {', '.join(column_defs)}
+        )
+        """
+    )
+    conn.execute(
+        f"""
+        INSERT INTO api_keys_new ({', '.join(current_columns)})
+        SELECT {', '.join(current_columns)}
+        FROM api_keys
+        """
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)